Dockerfile

```diff
@@ -11,7 +11,11 @@ RUN pip install --upgrade pip
 
 WORKDIR /builder
 
-COPY src/backend/marsha src/backend/setup.* /builder/
+COPY src/backend/setup.* /builder/
+
+# We need to preserve the `marsha` directory for setup.cfg
+RUN mkdir /builder/marsha/
+COPY src/backend/marsha/ /builder/marsha/
```

Comment:
I deploy on staging an image built without these 2 lines and it works fine. So you can remove them :+1: